source/addonHandler/__init__.py

```diff
@@ -737,7 +739,8 @@ class AddonManifest(ConfigObj):
 # Name of the author or entity that created the add-on
 author = string()
 
-# Version of the add-on. Should preferably in some standard format such as x.y.z
+# Version of the add-on.
+# Should be in <major>.<minor>.<patch> format.

```

Comment:
Does it mean that in the future, X.Y.Z will be mandatory?
For now, I have not dealt with beta versions. However, I was hoping to use version names such as "4.3beta1". "4.3.1" does not indicate clearly to the user that it is a beta version. Having a version name indicating clearly which version is a beta and which not in the add-on manager (or in the future add-on store) would be a plus IMO.

Same for dev versions even if I personally do not plan to release dev versions of my add-ons.

Reply:
submissions with these formats may not be valid for the add-on store. The add-on store requires a major.minor.pitch version scheme.
This current PR just proposes encouraging authors to follow the standards for the add-on store when making a manifest, not force any mandatory changes.

It's still an open question whether or not we add validation enforcing major.minor.patch in all add-on manifests, not just submissions to the add-on store.
This would not be until a breaking release of course.

Reply:
```suggestion
# Suggested convention is <major>.<minor>.<patch> format.
```